lab-alex-bessie-david/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const reader = require(`${__dirname}/lib/reader.js`);
+const transformer = require(`${__dirname}/lib/transformer.js`);
+const writer = require(`${__dirname}/lib/writer.js`);
+
+// node index.js './data/palette-bitmap.bmp' './data/newpalettebitmap.bmp' 'blackout'
+reader(process.argv[2], function (err, data) {
+  let buff = transformer(data, process.argv[4]);
```

Comment:
Consider that your transform function could potentially throw, so it might be worth wrapping the call in a try/catch and handling the errors more elegantly, rather than just blasting the console with the error and letting the app crash.